### PR TITLE
DPDK: fix meson installation for SUSE, RHEL & CentOS distros.

### PR DESCRIPTION
**Pull request by abhimarathe**
```
[LISAv2 Test Results Summary]
Test Run On           : 08/24/2020 18:09:15
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:16

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 DPDK                 VERIFY-DPDK-BUILD-AND-TESTPMD-TEST                                                PASS                15.57
      ARMImageName: SUSE SLES-15-sp1 gen1 latest, Networking: SRIOV, TestLocation: eastus2, Initial Kernel Version: 4.12.14-8.33-azure


Logs can be found at C:\LISAv2\HD33\TestResults\2020-08-24-11-08-31-0753
[LISAv2 Test Results Summary]
Test Run On           : 08/24/2020 18:39:46
ARM Image Under Test  : RedHat : RHEL : 7.5 : latest
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:13

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 DPDK                 VERIFY-DPDK-COMPLIANCE                                                            PASS                10.06
      ARMImageName: RedHat RHEL 7.5 latest, Networking: SRIOV, TestLocation: eastus2, Initial Kernel Version: 3.10.0-862.43.1.el7.x86_64


Logs can be found at C:\LISAv2\UL37\TestResults\2020-08-24-11-39-03-5836
```

### CODE REVIEW

File: Testscripts/Linux/dpdkUtils.sh [hunk 1]
```diff
@@ -318,8 +324,11 @@ function Install_Dpdk () {
 	LogMsg "MLX_PMD flag enabling on ${1}"
 	if type Dpdk_Configure > /dev/null; then
 		echo "Calling testcase provided Dpdk_Configure(1) on ${1}"
-		ssh ${1} "cd ${LIS_HOME}/${DPDK_DIR} && meson build"
-		#ssh ${1} "sed -ri 's,(MLX._PMD=)n,\1y,' ${LIS_HOME}/${DPDK_DIR}/build/.config"
+		if [[ ${DISTRO_NAME} == rhel || ${DISTRO_NAME} == centos ]]; then
+			ssh ${1} "cd ${LIS_HOME}/${DPDK_DIR} && PATH=$PATH:/opt/rh/rh-python36/root/usr/bin/ && meson build"
```
Comment (juhlee-microsoft) [inline Testscripts/Linux/dpdkUtils.sh:328]: If we are moving forward to python38 or later. this path is not valid

**Review by juhlee-microsoft — CHANGES_REQUESTED** [review]

File: Testscripts/Linux/dpdkUtils.sh (hunk 1 above)
Comment (abhimarathe) [reply Testscripts/Linux/dpdkUtils.sh:328]: For DPDK installation meson version 0.47.1 is [required ](http://doc.dpdk.org/guides/prog_guide/build-sdk-meson.html)which comes with python36 minimum in RHEL. That's why I included it.

**Comment by juhlee-microsoft** [comment]
@abhimarathe Are you still working on this PR?

**Comment by abhimarathe** [comment]
Closing this draft as #1010  will cover this PR.

**Pull request closed without merging**